Dockerfile

```diff
@@ -22,3 +22,5 @@ RUN poetry install
 
 FROM base as prod
 RUN poetry install --no-dev
+
+CMD ['flask', 'db', 'upgrade']
```

Comment:
I would not do it here. this is like a preparation before deploying the app, not the build instruction. your build will fail if there's no database. this should be invoked by docker-compose.

Reply:
Done